moveit_ros/visualization/motion_planning_rviz_plugin/src/ui/motion_planning_rviz_plugin_frame.ui

```diff
@@ -704,59 +756,149 @@
+              <property name="sizePolicy">
+               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
+                <horstretch>0</horstretch>
+                <verstretch>0</verstretch>
+               </sizepolicy>
+              </property>
+              <property name="minimumSize">
+               <size>
+                <width>30</width>
+                <height>0</height>
+               </size>
+              </property>
+              <property name="toolTip">
+               <string>Allows the robot to replan if the plan to be executed collides with the environment.</string>
+              </property>
               <property name="text">
-               <string>Use Cartesian Path</string>
+               <string>Replanning</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
-             <widget class="QCheckBox" name="collision_aware_ik">
+             <widget class="QCheckBox" name="allow_looking">
+              <property name="sizePolicy">
+               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
+                <horstretch>0</horstretch>
+                <verstretch>0</verstretch>
+               </sizepolicy>
+              </property>
+              <property name="minimumSize">
+               <size>
+                <width>30</width>
+                <height>0</height>
+               </size>
+              </property>
+              <property name="toolTip">
+               <string>Allows the robot to reposition its sensor to look around. A sensor needs to be set up and the planning algorithm has to support this, otherwise it has no effect.</string>
+              </property>
               <property name="text">
-               <string>Use Collision-Aware IK</string>
+               <string>Sensor Positioning</string>
               </property>
               <property name="checked">
-               <bool>true</bool>
+               <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
-             <widget class="QCheckBox" name="approximate_ik">
+             <widget class="QCheckBox" name="allow_external_program">
+              <property name="sizePolicy">
+               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
+                <horstretch>0</horstretch>
+                <verstretch>0</verstretch>
+               </sizepolicy>
+              </property>
+              <property name="minimumSize">
+               <size>
+                <width>30</width>
+                <height>0</height>
+               </size>
+              </property>
+              <property name="toolTip">
+               <string>Allow communication with an external program that sends an interactive marker position to control the robot.</string>
```

Comment:
```suggestion
               <string>Allow control of start and end poses as well as triggering of planning (and execution?) from external programs via ROS topics.</string>
```

Reply:
Ah I see, that's what happens. It looks like it only makes the plugin subscribe to the interactive marker topics with [this callback function](https://github.com/ros-planning/moveit/blob/master/moveit_ros/robot_interaction/src/robot_interaction.cpp#L674), but that doesn't trigger a plan or execution. I changed the text, please have a look.

Reply:
The checkbox triggers the following function, which also enables topics to trigger planning and execution:
https://github.com/ros-planning/moveit/blob/91e1f095c2e68187e19f975951011c20f0d8b51e/moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame.cpp#L257-L279